manifests/0000_30_cluster-api_00_credentials-request.yaml

```diff
@@ -76,8 +76,69 @@ spec:
   providerSpec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: AzureProviderSpec
-    roleBindings:
-    - role: Contributor
+    permissions:
```

Comment:
In order for `ccoctl` to create a user-assigned managed identity in Azure the `CredentialsRequest` will need to list `serviceAccountNames` as with the other platform `CredentialsRequests`

Reply:
Done.